### ISSUE TITLE
feat: add taints to the node on creation

### ISSUE THREAD

**Issue by mumoshu** [weaveworks/eksctl#396]
As @whereisaaron suggested in another issue at https://github.com/weaveworks/eksctl/issues/394#issuecomment-451508249, this is necessary to avoid misplacing pods after the kubelet created the node resource but before the user adds a taint(whose purpose is to avoid the pods being scheduled!)

kubelet provides the `--taints` option to adds taints on node creation, which completely resolves this problem.

I'd suggest adding `--taints` or `--node-taints` to the command-line flags for `create cluster` and `create nodegroup`, and possibly in another PR `taints` or `nodeTaints` in another PR after #381 lands.

**Comment by aparamon** [comment]
It is proposed to provide a more general feature of passing arbitrary [kubelet options](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options), similar to 
[EKS BootstrapArguments](https://docs.aws.amazon.com/en_us/eks/latest/userguide/getting-started.html#eks-launch-workers).

Specifying taints would indeed be very useful, in order to implement dedicated Load Balancer nodes:
https://github.com/weaveworks/eksctl/issues/440.

**Comment by errordeveloper** [comment]
It'd be good to make use this change as an opportunity to switch over from flags to kubelet config. We already have config, we just need to start generating and move labels and some of the other flags into it.

**Comment by errordeveloper** [comment]
There had been a few requests for this, so I might start working on this for the next release.

**Comment by errordeveloper** [comment]
We should switch to kubelet config files (https://github.com/weaveworks/eksctl/issues/156) before implementing this.
